### DISCUSSION TITLE
Can we speed up recurring scrapes like keeping the browser open in BG

### DISCUSSION THREAD

**Discussion by MrMegamind** [omkarcloud/google-maps-scraper#89] [Q&A]
It currently takes around 20 seonds for result for each request
Is there a way to keep browser running in bg so recurring requests can be speed up

**Comment by Chetan11-dev** [comment]
Thank you, @MrMegamind, for opening the issue and for using our Google Maps Scraper Tool.
I am not sure about your question. Could you please clarify?
We already run chrome in background.
